atlp-rwanda/atlp-pulse-mobile
# implementation of Attendance Page

### What does this PR do?
[x] Allowing trainee to see attendance
[x] Fixing ratings and performance

### How should this be manually tested?
To test this you need to clone this project and switch to `attendance-page` branch, install all dependencies `npm install` then start the project `npm start`.
On your app first Login before taking atendance. use web and take attendance for that trainee and check if you are able to see recorded attendance

To test ratings and performance, you have to make sure trainee is rated by whether ttl or coordinator from web app then login on the mobile to see ratings and performance
### Screenshots

<img width="180" src="https://github.com/user-attachments/assets/368261de-e578-4bd0-9eeb-a3a6dd4f4e97" />
<img width="180" src="https://github.com/user-attachments/assets/7b9bff64-b9f2-4df4-a178-fe1058d74c20" />
<img width="180" src="https://github.com/user-attachments/assets/04ba881d-1fa7-436b-b5be-5876992c0eca" />
<img width="180" src="https://github.com/user-attachments/assets/25fec699-97f4-481a-b57e-14f117ae7905" />
